README.md

```diff
@@ -157,7 +157,7 @@ outgoing requests using all the configured propagator formats.
 
 | System property  | Environment variable | Description                                                                                                     |
 |------------------|----------------------|-----------------------------------------------------------------------------------------------------------------|
-| otel.propagators | OTEL_PROPAGATORS     | Default is `tracecontext` (W3C). Other supported values are `b3`, `b3single`, `jaeger`, `ottracer`, and `xray`. |
+| otel.propagators | OTEL_PROPAGATORS     | Default is `tracecontext` (W3C). Other supported values are `b3`, `b3multi`, `jaeger`, `ottracer`, and `xray`. |
```

Comment:
The behavior of the b3 propagators is a bit confusing so maybe needs some additional docs

```
`b3` and `b3multi` both extract from the same set of headers, x-b3 multi format and b3 single format. But `b3` injects as b3 single, and `b3multi` injects as x-b3 multi. If you set both `b3` and `b3multi` you can accept and inject all types of b3 headers.